airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/uploader/config/UploaderConfig.java

```diff
@@ -35,11 +31,11 @@ public class UploaderConfig {
   private BigQuery bigQuery;
   private Map<UploaderType, BigQueryRecordFormatter> formatterMap;
   private boolean isDefaultAirbyteTmpSchema;
+  private String datasetLocation;
+  private boolean gcsUploadingMode;
```

Comment:
This seems to be a remnant when this code path was exercised for standard and GCS. now they are split with this as always false.